Sources/WebUI/WebView.swift

```diff
@@ -108,6 +109,17 @@ public struct WebView {
         return modified
     }
 
+    /// Sets the scale factor by which the web view scales content.
+    /// - Parameter factor: A fraction greater than 0.0 you use to sepecify the page scale.
```

Comment:
I think "sepecify" is a typo for "specify".
```suggestion
    /// - Parameter factor: A fraction greater than 0.0 you use to specify the page scale.
```